modules/distr-tracing-tempo-getting-the-tempostack-instance-logs.adoc

```diff
@@ -0,0 +1,17 @@
+// Module included in the following assemblies:
+//
+// * observability/distr_tracing/distr_tracing_tempo/distr-tracing-tempo-troubleshooting.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="getting-tempo-stack-logs_{context}"]
+= Getting the TempoStack instance logs
+
+You can get the logs from a TempoStack component as follows.
+
+.Procedure
+
+. Locate the component you want to get the logs for. You can do this by listing all the deployments on an specific namespace.
+
+. Identify the pods that belong to this component.
+
+. Watch the logs by using `oc logs` command or retrieve the logs by using the web console.
```

Comment:
Should include an example command here instead I think, something like:

```
. Watch the logs for the component by running the following command:
+
[source,terminal]
----
$ oc logs <component>
----
+
You can also view logs by navigating to X in the {product-title} web console.
```
(I'm not 100% what the command would be, this is just an example)
I think maybe there should be separate procedures also for using the `oc` CLI versus using the web console - neither one is listed here as `.Prerequisites` even though a user would need either access to the web console or to have the `oc` CLI installed here to follow this procedure.

Reply:
@rubenvp8510, two requests to you here:

1. Could you recommend a specific `oc logs <options>` command?

2. What's the navigation path in the web console?